### PR TITLE
Clean up the cocoapod installation

### PR DESCRIPTION
**Pull request by kcw-grunt**
- Remove Firebase pod
- Added Firebase Swift package
- Removed  some old dead code

### CODE REVIEW

File: loafwallet.xcworkspace/xcshareddata/swiftpm/Package.resolved [hunk 1]
```diff
@@ -1,6 +1,60 @@
 {
   "object": {
     "pins": [
+      {
+        "package": "abseil",
```
Comment (mosadialiou) [inline loafwallet.xcworkspace/xcshareddata/swiftpm/Package.resolved:5]: where is used `abseil` lib?
This question goes for all other packages except the `Firebase` package

**Review by mosadialiou — CHANGES_REQUESTED** [review]

File: loafwallet.xcworkspace/xcshareddata/swiftpm/Package.resolved (hunk 1 above)
Comment (kcw-grunt) [reply loafwallet.xcworkspace/xcshareddata/swiftpm/Package.resolved:5]: Otherwise we are good.  Right : @mosadialiou ?

Comment (mosadialiou) [reply loafwallet.xcworkspace/xcshareddata/swiftpm/Package.resolved:5]: OK. So yes we are good

File: loafwallet.xcodeproj/xcshareddata/xcschemes/loafwallet.xcscheme [hunk 2]
```diff
@@ -72,6 +72,12 @@
             ReferencedContainer = "container:loafwallet.xcodeproj">
          </BuildableReference>
       </BuildableProductRunnable>
+      <CommandLineArguments>
+         <CommandLineArgument
+            argument = "-FIRAnalyticsDebugEnabled"
+            isEnabled = "YES">
```
Comment (mosadialiou) [inline loafwallet.xcodeproj/xcshareddata/xcschemes/loafwallet.xcscheme:78]: In prod releases this flag is set to FALSE, right?

**Review by mosadialiou — CHANGES_REQUESTED** [review]

File: loafwallet.xcodeproj/xcshareddata/xcschemes/loafwallet.xcscheme (hunk 2 above)
Comment (kcw-grunt) [reply loafwallet.xcodeproj/xcshareddata/xcschemes/loafwallet.xcscheme:78]: Good point let me change that.

**Review by mosadialiou — APPROVED** [review]
LGTM

**Pull request merged**